packages/website/ts/components/dialogs/ledger_config_dialog.tsx

```diff
@@ -282,7 +282,10 @@ export class LedgerConfigDialog extends React.Component<LedgerConfigDialogProps,
         if (didSucceed) {
             this.setState({
                 stepIndex: LedgerSteps.SELECT_ADDRESS,
+                connectionErrMsg: '',
             });
+        } else {
+            this.props.dispatcher.updateBlockchainIsLoaded(true);
```

Comment:
Hm, this seems like an odd place for this. Mind explaining or adding a `// HACK` comment if this is one? If we fail to fetch the addresses and balances, we set blockchainIsLoaded to `true`?

Reply:
Yea now that you mention it the behavior isn't as it should be. It starts loading immediately when you click to switch to ledger (which is why I opted to remove the loading state for failures, or there is no way to get out of it)

Reply:
ok, either let's refactor or leave a comment explaining.